eva-accession-import/src/test/resources/application.properties

```diff
@@ -14,4 +14,9 @@ parameters.chunkSize=100
 parameters.forceRestart=false
 parameters.pageSize=100
 
+spring.data.mongodb.database=dbsnp-import-test
+spring.data.mongodb.host=localhost
+spring.data.mongodb.password=
+spring.data.mongodb.port=27017
```

Comment:
I think it would be preferable to move these Mongo properties to a separate properties file because:
* They are specific to a single test
* Said test doesn't need the rest of the properties

The database could then be renamed like that test, to avoid clashes if/when others decide to use nosqlunit too.

Reply:
the other points are valid, but that test does need the properties for the datasource and the parameters to be able to load the spring application context to use the NoSqlUnit and mongoTemplate.